### PR TITLE
home-assistant: 0.111.0 -> 0.111.4

### LINKED ISSUE #90149: nixos/home-assistant: test failure
```
hass # Connection to localhost 8123 port [tcp/*] succeeded!
(11.48 seconds)
hass: must succeed: curl --fail http://localhost:8123/lovelace
(0.07 seconds)
hass: must succeed: curl --fail -H 'x-ha-access: some_secret' http://localhost:8123/api/
hass # [   25.987891] hass[751]: 2020-06-11 14:06:25 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 127.0.0.1
hass # curl: (22) The requested URL returned error: 401 Unauthorized
hass: output: 
Test "Check that Home Assistant's web interface and API can be reached" failed with error: "command `curl --fail -H 'x-ha-access: some_secret' http://localhost:8123/api/` failed (exit code 22)"
```

### PR DESCRIPTION
**Pull request by Mic92**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Mic92** [comment]
@GrahamcOfBorg test home-assistant

File: pkgs/servers/home-assistant/frontend.nix [hunk 1]
```diff
@@ -4,11 +4,11 @@ buildPythonPackage rec {
   # the frontend version corresponding to a specific home-assistant version can be found here
   # https://github.com/home-assistant/home-assistant/blob/master/homeassistant/components/frontend/manifest.json
   pname = "home-assistant-frontend";
-  version = "20200603.2";
+  version = "20200617.0";
```
Comment (mweinelt) [inline pkgs/servers/home-assistant/frontend.nix:7]: 0.111.4 uses an older frontend version: https://github.com/home-assistant/core/blob/0.111.4/homeassistant/components/frontend/manifest.json#L5

Comment (Mic92) [reply pkgs/servers/home-assistant/frontend.nix:7]: I have to fix my upgrade script...

**Comment by Mic92** [comment]
@GrahamcOfBorg test home-assistant

**Comment by Mic92** [comment]
I published an update script as well.

**Comment by Mic92** [comment]
Mhm. That eval issue looks unrelated to mine.

**Comment by jonringer** [comment]
@GrahamcOfBorg eval

**Comment by FRidh** [comment]
```
derivation 'homeassistant-0.111.4' has invalid meta attribute 'tests'
```

File: pkgs/servers/home-assistant/default.nix [hunk 2]
```diff
@@ -128,12 +123,16 @@ in with py.pkgs; buildPythonApplication rec {
...
+      inherit (nixosTests) home-assistant;
+    };
   };
 
   meta = with lib; {
     homepage = "https://home-assistant.io/";
     description = "Open-source home automation platform running on Python 3";
     license = licenses.asl20;
     maintainers = with maintainers; [ dotlambda globin mic92 ];
+    tests.nixos = nixosTests.fish;
```
Comment (FRidh) [inline pkgs/servers/home-assistant/default.nix:136]: This should be in the `passthru`.

**Review by mweinelt — APPROVED** [review]
Deployed and tested. LGTM.

Unrelated nit: The met.no weather component is still not working :cry:

**Pull request merged**